services/auth/auth-supabase.service.ts

```diff
@@ -0,0 +1,7 @@
+import { createClient } from '@supabase/js';
```

Comment:
Create a class for it , and add the functions

The class shall follow an interface (so tomorrow we can implement different types of services)